common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -3392,11 +3392,19 @@ public static enum ConfVars {
     MERGE_CARDINALITY_VIOLATION_CHECK("hive.merge.cardinality.check", true,
       "Set to true to ensure that each SQL Merge statement ensures that for each row in the target\n" +
         "table there is at most 1 matching row in the source table per SQL Specification."),
+    SPLIT_UPDATE("hive.split.update", true,
```

Comment:
I understand why this is true for testing, but I would guess that we set this `false` by default to keep the original behaviour

Reply:
It can be changed back to false. I just enabled it for testing purposes. Once this patch ready to submit I will disable it.